manual/owner.md

```diff
@@ -0,0 +1,233 @@
+  
+You have to **connect each item you create to an organisation**. You can use the **default organisation** for all the items. If your organisation has many departments or faculties that have their own resources, we recommend creating separate organisations for each department.
+
+![Owner process](img/rems_owner.png)
+
+:exclamation: Note! You have to create all the other items before creating a catalogue item.
+
+You can also use the elements that you have created earlier.
+
+These are just general instructions and recommendations, note that your home organisation’s practices might be different.
+
+### 1. Create an application form
+
+1. Log in to REMS and navigate to Administration>Forms
+![Forms](img/forms.png)
+2. Select **Create form**.
+3. Name your form. Give the form a descriptive name so that you can find it easily later. The name will show in the list view.
+4. Add the fields you want to have in the application. The preview on the right side shows how an applicant would see the application form.
+5. Select **Save** when you are finished with the form.
+
+:exclamation: Note! You can only edit the form as long as it has not been connected to a catalogue item.
+
+Application forms are totally customisable. You can create different fields, such as free text or multiple choice, and add them in any order you want. Create the form in the way that best suits you and your organisation.
+
+#### Form fields explained
+
+**Application title**: Use this field if you want the applicants to provide a name for their application.
+**Option list**: Use option list if you want the applicant to choose one of the given options. ID-code is the identifier for the option. It is shown, for example in the reports, but the applicant will only see the label fields. For example: You can put “Y” as an ID-code for Yes-option and “N” for No-option.
+**Multi-select list**: Use a multi-select list if you want the applicant to choose one or more of the given options.
+**Email address**: Email address field will validate the email the applicant has provided.
+**Attachment**: Applicant can add one or multiple attachments.
+**Label**: Use labels as descriptive texts if you want to give the applicant additional information, such as instructions, regarding some field.
+**Header**: Consider using headers if you want to divide the application form into sections.
+
+Placeholder texts: It is possible to add placeholder text to most of the fields. Use placeholder texts to provide an example on how the field should be filled. Placeholders help the applicant to fill in the field correctly. For example:
+
+![Placeholder text](img/placeholder.png)
+
+:exclamation: Note! **Do not give important information, such as instructions, as placeholder texts** because it will be difficult for the applicant to read and it is not accessible. Use labels or info text if you want to give instructions how to fill in a field.
```

Comment:
hmm ok info text is mentioned here but could be highlighted better

Reply:
Since the UI has actually changed quite a bit I'll leave this as it is now and update it later with up-to-date screen shots.